docs/guides/advanced/alternative-clients.md

```diff
@@ -49,6 +49,48 @@ exporter = OTLPSpanExporter(
 )
 ```
 
+## Example with NodeJS
+
+Create these files:
```

Comment:
I would rather use the following shell command:
```
npm init es6 -y # creates package.json with type module
npm install @opentelemetry/sdk-node
```

Reply:
Thanks